15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,20 @@
+const totalIntegers = function(obj, count = 0) {
+
+  if (typeof obj !== 'object' || obj === null) {
```

Comment:
What's your opinion on extracting this out to an `isObject` helper function, since there isn't a built-in one in JS?
e.g.
```javascript
const isObject = function(value) {
  return typeof value === 'object' && value !== null;
}
```
Then we can do 
`if (!isObject(obj))` and later, `else if (isObject(el))`.

That can also help drive it home that object checking has to involve a null check.

Reply:
Sure, lets dry up the code a bit